bp/index.html

```diff
@@ -1295,6 +1305,7 @@ <h4 class="subhead">Why</h4>
           <section class="outcome">
             <h4 class="subhead">Intended Outcome</h4>
             <p>Spatial Things become part of the Web’s global information space enabling them be linked with other <a>Spatial Things</a> and other resources and for those <a>links</a> to be durable. In other words, <a>spatial data</a> becomes part of the Web of Data.</p>
+		  <p>Your data is more <a href="https://www.go-fair.org/fair-principles/">findable</a></p>
```

Comment:
I don't believe this link is necessary.  There are other labelled links that lead to this web site, so this is redundant.